src/load.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+import csv
+import itertools
+import pickle
+import os
+
+from pprint import pprint
+import gensim
+from gensim.models import Phrases
+import spacy
+from spacy.lang.en.stop_words import STOP_WORDS
+from smart_open import open
+from pprint import pprint
+
+from config import Config
+import time
+
+def split(file_in, dir_out, train_size=34000):
+    """ Splits dataset to train and test portions
+
+    Args:
+        file_in: raw dataset file path
+        dir_out: path to dir for output
+        train_size: size of train portion+1
+
+    """
+    with open(file_in) as f:
+        x = csv.reader(f)
+        with open(os.path.join(dir_out,'train_wiki.csv'), 'w') as f:
+            for z in range(34000):
+                r = next(x)
+                w = csv.writer(f)
+                w.writerow(r)
+
+        with open(os.path.join(dir_out,'test_wiki.csv'), 'w') as f:
+            for r in x:
+                w = csv.writer(f)
+                w.writerow(r)
+
```

Comment:
tomuhle nerozumim. Proc  pro kazdy radek v `x` vytvorite novy csv writer `w` a pak do nej ten jeden radek zapisete?

Reply:
to je chyba.